pkgs/data/fonts/victor-mono/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, fetchzip }:
+
+let
+  version = "1.2.1";
```

Comment:
feel like version could be moved below, it's already a recursive set.

Reply:
Wargarbl, I lost this bouncing between whether I could/should make this fixed-output or not--but yes I'll fix those right away, sorry about that xD :).

Reply:
Unfortunately appears fetchzip doesn't allow ("expect") the "version" attribute which is why it's hoisted above.  Unsurprisingly it doesn't take `pname` either :).

Reply:
ah, disregard then. :)

Reply:
Agreed, and a fetchzipWithExtras or something seems unsatisfying as well. At least for sole purpose of avoiding the `let version = ""; in` pattern used as the alternative.  Whatcha gonna do.

Anyway, look good to folks? The double-extract is for the sake of starting from a versioned tarball :).